megamek/src/megamek/client/ui/swing/tooltip/UnitToolTip.java

```diff
@@ -1882,7 +1889,7 @@ private static StringBuilder c3Info(Entity entity, boolean details) {
                 sC3Info += "<BR>";
             }
 
-            result = guiScaledFontHTML(uiC3Color(), -0.2f) + sC3Info + "</FONT>";
+            result = guiScaledFontHTML(GUIP.getUnitToolTipHighlightColor(), -0.2f) + sC3Info + "</FONT>";

```

Comment:
What about the C3 color? Or is that the same as the unit tooltip highlight color now?

Reply:
c3 and the highlight color were similar colors.  I did not want to add a lot of extra color settings that are only used in one spot, but if you think it should have one, I can add it.

Reply:
Nah, it's fine for now.